### DISCUSSION TITLE
Feature request: salt-cloud oVirt/RHEV platform management support

### DISCUSSION THREAD

**Discussion by skandyla** [saltstack/salt#67655] [Feature Requests]
Hi folk,
it would be great to have oVirt (https://en.wikipedia.org/wiki/OVirt) platform support at salt.
Ovirt in two words something like Vmware vcenter.

Ansible, for instance, has such feature.
http://docs.ansible.com/ansible/ovirt_module.html

**Comment by damon-atkins** [comment]
+1

**Comment by jfindlay** [comment]
@skandyla, thanks for the feature request.

**Comment by sldenazis** [comment]
+1

**Comment by cmeisinger** [comment]
+1

**Comment by allierichards** [comment]
+1

**Comment by grandmasterv** [comment]
+1

**Comment by wing731** [comment]
+1

**Comment by mpetersen42** [comment]
+1

**Comment by lgogolin** [comment]
+1

**Comment by ernstae** [comment]
A +1 from me too!

**Comment by mykaul** [comment]
Updated link to Ansible's oVirt modules (which have been re-written, expanded and fully documented):
https://github.com/ansible/ansible/tree/devel/lib/ansible/modules/cloud/ovirt

**Comment by marawu** [comment]
+1

**Comment by fabfurnari** [comment]
+1

**Comment by tjgruber** [comment]
Status of this?
Would be nice to have.

**Comment by dsmoljanovic** [comment]
Any information?

**Comment by ggruner** [comment]
+1

**Comment by deankannenberg** [comment]
+1
Is there any plan to apply this request?

**Comment by lucawerner** [comment]
+1
